src/routes/apiDirectory.js

```diff
@@ -19,6 +20,7 @@ const router = express.Router();
 
 router.use(authMiddleware.unless({ path: [join('/api', loginPath)] }));
 
+router.use('/admin/users', adminUsersRouter);
```

Comment:
Why did this get added? It's also included on line 18 of the regular `adminRouter`

Reply:
Good catch. Removed.